src_lockfree/spsc_queue.ml

```diff
@@ -48,7 +48,27 @@ let push { array; head; tail; mask; _ } element =
     Array.set array (tail_val land mask) (Some element);
     Atomic.set tail (tail_val + 1))
 
+let[@inline] try_push t element =
+  try
+    push t element;
+    true
+  with Full -> false
```

Comment:
Here I try the `[@inline]`. But I don't think this is the right way to use it. Any help will be welcomed.